lib/kebab_van.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'take_away'
+
+class KebabVan < TakeAway
+  attr_accessor :menu, :time, :name
+
+  def initialize(name)
```

Comment:
Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.